srcbmi/mf6bmi.f90

```diff
@@ -833,7 +816,7 @@ function get_model_name(grid_id) result(model_name)
       end if
     end do
     
-    ! TODO_MJR: error message, should never get here...
+    write(*,*) 'BMI error: no model for grid id \"', grid_id, '\"'

```

Comment:
You should probably use the standard `sim_message` subroutine in the `GenericUtilitiesModule` module to write this message.